### PR TITLE
TorchTune --> torchtune

### PR DESCRIPTION
**Pull request by joecummings**
#### Context

This was annoying me.

#### Changelog
* See title

#### Test plan

1) Eyes
2) Unit tests for catching specific naming changes

### CODE REVIEW

**Review by ebsmothers — APPROVED** [review]
I thought we had done this refactor already?

**Review by kartikayk — COMMENTED** [review]
🔥 🔥

File: docs/source/conf.py [hunk 1]
```diff
@@ -84,7 +84,7 @@
 # Get TORCHTUNE_VERSION_DOCS during the build.
 torchtune_version_docs = os.environ.get("TORCHTUNE_VERSION_DOCS", None)
 print(f"torchtune_version_docs: {torchtune_version_docs}")
-project = "TorchTune"
+project = "torchtune"
```
[the remark this answers is not in the document]
Comment (joecummings) [reply docs/source/conf.py:87]: Likely not - not ready to dig into what code is used and not-used in our docs setup just yet

[the remark this answers is not in the document]
Comment (ebsmothers) [reply docs/source/conf.py:87]: OK yeah just wanna make sure this isn't some sphinx magic that's gonna mysteriously break a link on the backend somewhere

**Pull request merged**